kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/OperatorChangeDetectionST.java

```diff
@@ -34,13 +35,15 @@
 import io.kroxylicious.systemtests.k8s.KubeClient;
 import io.kroxylicious.systemtests.templates.kroxylicious.KroxyliciousFilterTemplates;
 
+import static io.kroxylicious.systemtests.TestTags.OPERATOR;
 import static io.kroxylicious.systemtests.k8s.KubeClusterResource.kubeClient;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 
 /**
  * The Kroxylicious system tests.
  */
+@Tag(OPERATOR)
```

Comment:
I've read about tags but never used them in anger. Where does this get picked up and used?

Reply:
It is currently used in our CI for release to avoid the kafka client loop for those who don't need it.

But tags are really useful for other matters like regression, to mark those tests that we want to run.